ci/infra/testrunner/tests/test_skuba_upgrade.py

```diff
@@ -8,7 +8,7 @@
 
 @pytest.fixture
 def setup(request, platform, skuba):
-    platform.provision(num_master=1, num_worker=1)
+    platform.provision(num_master=3, num_worker=3)
```

Comment:
you could also drop the args, as it should be the default, but it's a minor detail

Reply:
I would prefer to have the args there, so it's obvious to the reader/code-reviewer/qa how many workers and masters are needed.